### PR TITLE
Offline DQM L1T Muon CMSSW_8_1_X

### PR DESCRIPTION
**Pull request by astakia**
Contains update of the Offline DQM L1T Muon efficiency code with the 2016 L1T variables (Stage2 Muons implementation). 

The already made 8_0_X version of this pull request can be found here: PR #16233

### CODE REVIEW

File: DQMOffline/L1Trigger/python/L1TEfficiencyHarvesting_cfi.py [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-    cms.untracked.PSet( dqmBaseDir = cms.untracked.string("L1T/Efficiency/Muons"),
-                        plots      = cms.untracked.vstring("EffvsPt16","EffvsEta16","EffvsPhi16",\
-                                                           "EffvsPt20","EffvsEta20","EffvsPhi20",\
-                                                           "EffvsPt25","EffvsEta25","EffvsPhi25")
-                      )
-  )
-                                         
+    verbose  = cms.untracked.bool(False),
+    plotCfgs = cms.untracked.VPSet(
+        cms.untracked.PSet( dqmBaseDir = cms.untracked.string("L1T/Efficiency/Muons"),
```
Comment (dmitrijus) [inline DQMOffline/L1Trigger/python/L1TEfficiencyHarvesting_cfi.py]: DO NOT use tabs inside python files. Use 4 spaces.
Also, https://www.python.org/dev/peps/pep-0008/

**Review by dmitrijus — COMMENTED** [review]
Besides the count less space/tab errors, the code looks fine (also in python files, where space errors are more crtiical).

Please configure your editor to expand tabs into spaces when you press a 'tab' button.

**Comment by dmitrijus** [comment]
I will wait for L1 signature before signing myself.

**Comment by astakia** [comment]
@dmitrijus : tabs fixed (in the 8_0_X PR as well)

**Comment by dmitrijus** [comment]
Thanks, look fine!
But get L1 signature first - I will sign right after you do.

**Comment by mulhearn** [comment]
+1

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**